__tests__/UI-tests/WelcomeScreens-test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { render } from "@testing-library/react-native";
-import WelcomeScreens from "../app/index";
+import WelcomeScreens from "../../app/index";
```

Comment:
same goes here as in the comment of this file: __tests__/UI-tests/Forgot-Password-test.tsx